server_environment/serv_config.py

```diff
@@ -29,29 +29,35 @@
 
 from .system_info import get_server_environment
 
-from openerp.addons import server_environment_files
-_dir = os.path.dirname(server_environment_files.__file__)
-
-# Same dict as RawConfigParser._boolean_states
-_boolean_states = {'1': True, 'yes': True, 'true': True, 'on': True,
-                   '0': False, 'no': False, 'false': False, 'off': False}
-
-if not system_base_config.get('running_env', False):
-    raise Exception(
-        "The parameter 'running_env' has not be set neither in base config "
-        "file option -c or in openerprc.\n"
-        "We strongly recommend against using the rc file but instead use an "
-        "explicit config file with this content:\n"
-        "[options]\nrunning_env = dev"
-    )
+try:
+    from openerp.addons import server_environment_files
+except ImportError:
+    _dir = False
+else:
+    _dir = os.path.dirname(server_environment_files.__file__)
+
+if _dir:
+    # Same dict as RawConfigParser._boolean_states
+    _boolean_states = {
+        '1': True, 'yes': True, 'true': True, 'on': True,
+        '0': False, 'no': False, 'false': False, 'off': False}
+
```

Comment:
@dreispt We have to add a way to check if module is installed here.
There is not need to raise an exception if module is not installed. 

By default we wanted to ensure that setup is correct if module was installed and OpenERP should not boot if it is the case.